amundsen_application/api/utils/search_utils.py

```diff
@@ -21,6 +21,7 @@ def map_table_result(result: Dict) -> Dict:
         'description': result.get('description', None),
         'database': result.get('database', None),
         'schema': result.get('schema', None),
+        'schema_description': result.get('schema_description', None),
```

Comment:
Regarding the failing test, you will want to edit this file https://github.com/lyft/amundsenfrontendlibrary/blob/master/tests/unit/api/search/test_v0.py. The test is failing because now we expect the search endpoint to process a `schema_description`, but the key is missing in `MOCK_PARSED_RESULTS`. So you can either:
1. Add some dummy value for this like `'schema_description': 'test',` to `MOCK_TABLE_RESULTS`, and `MOCK_PARSED_RESULTS`
2. Add `'schema_description': None,` to `MOCK_PARSED_RESULTS`.